spec/lib/reports/careplus_exporter_spec.rb

```diff
@@ -245,7 +261,20 @@
     end
   end
 
-  include_examples "generates a report", :hpv
-  include_examples "generates a report", :menacwy
-  include_examples "generates a report", :td_ipv
+  context "with flu programme" do
+    let(:programme_type) { :flu }
+
+    it "uses FLUENZ code for nasal delivery" do
+      create_vaccination_record(delivery_method: :nasal_spray, session:)
+      code_index = headers.index("Vaccine Code 1")
+      row = parsed_csv[1]
+
+      expect(row[code_index]).to eq("FLUENZ")
+    end
+  end
+
+  it_behaves_like "generates a report", "hpv"
```

Comment:
Changed `include_examples` to `it_behaves_like ` because if for example `hpv` passes, the subsequent shared examples pass as well even though the specs should fail.

I believe `include_examples` simply includes the examples directly in the current context without additional nesting. With `it_behaves_like`, each programme type gets its own isolated test context, preventing potential state leakage between examples.

Reply:
Interesting, there are a few other places where we use `includes_examples`. I wonder if it's worth changing those too?

Reply:
I think so